File: tests/project_multilang/dart/expected/caller.dart
void run(Object c) {
  (c as ICallback).onReady();
  if ((c as ICallback).isEnabled()) {
    print("on");
  }
}

void multiline(bool flag) {
  bool isOneWay =
      false;
  use(isOneWay);
}

void use(bool b) { print(b); }

File: tests/project_multilang/java/expected/Caller.java
package demo;
public class Caller {
  void run(Object c) {
    ((ICallback) c).onReady();
    if (((ICallback) c).isEnabled()) {
      System.out.println("on");
    }
  }

  void multiline(boolean flag) {
    use(false);
  }

  void use(boolean b) { System.out.println(b); }
}
